src/main/java/uk/gov/companieshouse/model/dto/DissolutionGetResponse.java

```diff
@@ -0,0 +1,120 @@
+package uk.gov.companieshouse.model.dto;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.model.db.DissolutionGetDirector;
+import uk.gov.companieshouse.model.enums.ApplicationStatusEnum;
+import uk.gov.companieshouse.model.enums.ApplicationTypeEnum;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+public class DissolutionGetResponse {
+
+    private String ETag;
```

Comment:
Finding these properties difficult to read with annotations mixed in the middle. Try adding a line between each property for readability